src/test/java/edu/oakland/test/database/DatabasePersistentStorageStub.java

```diff
@@ -0,0 +1,42 @@
+package edu.oakland.test.database;
+
+import edu.oakland.helper.admin.LocationDataPoint;
+import edu.oakland.helper.admin.TrackData;
+import edu.oakland.production.database.DatabasePersistentStorage;
+
+public class DatabasePersistentStorageStub implements DatabasePersistentStorage {
+  int rfid = 10;
```

Comment:
Can we make this dynamically configurable? Below is one way to do it, or you could also add a setter.
```suggestion
  private int rfid;
  
  public DatabasePersistentStorageStub(int rfit) {
    this.rfid = rfid;
  }
```